src/main/drivers/timer.h

```diff
@@ -107,6 +111,26 @@ typedef struct timerHardware_s {
 #endif
 } timerHardware_t;
 
+#ifdef USE_TIMER_MGMT
+
+typedef uint8_t timerTag_t;
+
+typedef struct timerChannelConfig_s {
+    ioTag_t ioTag;
+#if defined(STM32F3) || defined(STM32F4) || defined(STM32F7)
+    uint8_t alternateFunction;
```

Comment:
Fair - though this is more an implementation tactical issue (nitty gritty) - looking more for a strategic comment / review on the approach.